### ISSUE TITLE
orphaned block on ethereum

### ISSUE THREAD

**Issue by tenlans** [coinfoundry/miningcore#466]
### Description
There are two blocks submitted by miner, but the status is orphaned on databases.
so I can not get any reward.
It is normal? or have other question ?
and my pool sharespersecond about 12 is too low ?

### Steps to Reproduce

1. server log
https://imgur.com/QeINNzX

2. block table
https://imgur.com/a/Xkz2s7x

### Reproduces how often
block 6660876: 2018-11-07 21:37:28
block 6673518: 2018-11-09 18:01:49

### Version
[Server]
Ubuntu 16.04
Latest miningcore as at Thu Oct 25 2018 from master
Parity v2.0.8-stable
Pool hashrate about 28GH/s
Pool sharespersecond about 12
Pool difficulty: 2G
[Client]
Windows 7 or Windows 10
EthDcrMiner64 v11.9

### Additional Information

"pools": [{
        "id": "eth",
        "enabled": true,
        "coin": "ethereum",
        "address": "0x",
        "rewardRecipients": [{
                "type": "op",
                "address": "0x",
                "percentage": 1.0
        }],
        "blockRefreshInterval": 500,
        "clientConnectionTimeout": 600,
        "banning": {
            "enabled": true,
            "time": 600,
            "invalidPercent": 50,
            "checkThreshold": 50
        },
        "ports": {
            "3333": {
                "listenAddress": "0.0.0.0",
                "difficulty": 0.5
            }
        },
        "daemons": [
            {
                "host": "127.0.0.1",
                "port": 8545
            }
        ],
        "paymentProcessing": {
            "enabled": true,
            "minimumPayment": 0.05,
            "minimumPaymentToPaymentId": 5.0,
            "payoutScheme": "PPLNS",
            "payoutSchemeConfig": {
                "factor": 2.0
            },
	"coinbasePassword": "xxxx",
            "keepUncles": false,
            "keepTransactionFees": false
        }
    }]

**Comment by HaffASE** [comment]
You should read this issue [https://github.com/paritytech/parity-ethereum/issues/9278] 
Most likely the problem is not in the miningcore but in the parity client

**Comment by tenlans** [comment]
@HaffASE thank you.
I try to use geth to sync blocks, but the miningcore always show the same log.

[2018-11-12 19:52:10.2889] [I] [eth] Daemons have downloaded 100.00% of blockchain from 199 peers 
[2018-11-12 19:52:15.2902] [I] [eth] Daemons have downloaded 100.00% of blockchain from 199 peers 
[2018-11-12 19:52:20.2940] [I] [eth] Daemons have downloaded 100.00% of blockchain from 199 peers


### Version
[Server]
Ubuntu 16.04
miningcore: v44
Geth 1.8.17-stable

**Comment by HaffASE** [comment]
@tenlans Show the Geth output to the request eth_syncing.
Did you turn on geth in mining mode? -mine
Аfter the version 1.8.14 it stopped turning on automatically, you can check it by request eth_gethWork

**Comment by oliverw** [comment]
@tenlans @HaffASE We are in a hot mess right now with Ethereum (not Ethereum Classic though) where we got Parity on one side which no longer reliably reports work to the pool. And then we got geth, which works once it finally synced but which often fails to complete syncing at all due to major performance problems. If syncing the full blockchain using geth works at all then only with an insanely fast SSD. That's the reason why Coinfoundry's ETH pool is still down.

**Comment by tenlans** [comment]
@HaffASE thanks again,  the geth need "-mine" option.
I had resoved the question, but get other log when I start to mine.

All shares are rejected: 
[2018-11-12 23:14:58.5540] [I] [eth] [0HLI8M4S5VBG3] Share rejected: stale share 
[2018-11-12 23:14:59.7237] [I] [eth] [0HLI8M4S5VBG2] Share rejected: stale share 
[2018-11-12 23:14:59.8556] [I] [eth] [0HLI8M4S5VBG3] Share rejected: stale share

and the last block number on the geth log is slowly the parity log.
they are diff 2 ~ 3 blocks

this is geth config: 
$ geth --mine --cache 4096 --rpc --maxpeers 200 console

I run "peerCount" command on geth console.
> net.peerCount
168

**Comment by tenlans** [comment]
@oliverw 
I use SSD to sync block with geth, but it still sync slowly.
Should I continue mine Ethereum with miningcore ? or switch other pool software ?

**Comment by oliverw** [comment]
@tenlans I'm not aware of any other open-source pool software supporting Ethereum.

**Comment by tenlans** [comment]
@oliverw 
Hi, What best solution do you think ? 
a. Parity + Miningcore 
b. Geth + Miningcore

I mine two orphaned block, so I try to switch parity to geth.
Could you help me ?

**Comment by HaffASE** [comment]
@oliverw I started mining in the Ropsten network on Parity. And I noticed that the miningcore does all blocks orphaned although it is not true. I get a reward for them. Here is an example

Miningcore logs 
2018-11-13 16:37:06.3819] [I] [eth1] Submitting block 4422473
[2018-11-13 16:37:06.4378] [I] [eth1] Daemon accepted block 4422473 submitted by 0xb1B5b8269D4FFA8813e98B39E77472E98bd9F879

psql record

14 | eth1   |     4422473 |        6622521085 | orphaned |      |   1 |  0.328507220252177 | 0xacd651468e5d7680bb96f7f4c6a29fbc32e7c44e9a57436680a8ced9599f82ab:0x4a3c80000434360c | 0xb1B5b8269D4FFA8813e98B39E77472E98bd9F879 | 0.000000000000 |        | 0xacd651468e5d7680bb96f7f4c6a29fbc32e7c44e9a57436680a8ced9599f82ab | 2018-11-13 16:37:06.381916

This block 4422473 has a wrong hash 0xacd651468e5d7680bb96f7f4c6a29fbc32e7c44e9a57436680a8ced9599f82ab, correct hash is [0xa139b0ef14f2852063807db84e5d6eba13e60e4caf5e40b309a2af0f7c3d8bf5](https://ropsten.etherscan.io/block/4422473)

Perhaps that is why all blocks become orphaned on pool stats
